src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -405,14 +408,14 @@ private void generateSummarySpreadsheet(
     }
 
     csvPrinter.print("Inactive Ballots Total");
-    // Undervotes should not be included in the Inactive Ballots count, even though we treat them
-    // as such internally. Subtract undervotes (which are static throughout a contest) from the
-    // inactive ballot totals.
-    BigDecimal numUndervotes =
-        roundTallies.get(1).getBallotStatusTally(StatusForRound.INACTIVE_BY_UNDERVOTE);
+    // DID_NOT_RANK_ANY_CANDIDATES should not be included in the Inactive Ballots count, even though
+    // we treat them as such internally. Subtract no-rankings (which are static throughout a
```

Comment:
What does this mean? That we treat no-rank ballots as inactive internally?

Reply:
Good call, this comment is outdated now that we have a separate "Did not rank" category.